### DISCUSSION TITLE
Section 2: Storage Factory under solidity imports

### DISCUSSION THREAD

**Discussion by ahpoh1349** [Cyfrin/foundry-full-course-cu#2382] [Q&A] [answered]
Patrick says need to do this method instead of the other if not later will cause us many problems,which way is it?

import {SimpleStorage, SimpleStorage2} from "./SimpleStorage.sol";   or using below better?
import "./SimpleStorage.sol";

is using the former better right?

**Comment by matcherbudz** [comment, marked as answer]
It is best to default using the named import with the curly brackets because it allows us to import specific functions, variables, or contracts from another file.
import {SimpleStorage, SimpleStorage2} from "./SimpleStorage"; is better

**Reply by ahpoh1349** [reply]
thanks Ser, appreciate it

**Comment by EngrPips** [comment]
Interesting.
